rel/vm.args

```diff
@@ -0,0 +1,30 @@
+## This file provide the arguments provided to the VM at startup
+## You can find a full list of flags and their behaviours at
+## http://erlang.org/doc/man/erl.html
+
+## Name of the node
+-name <%= release_name %>@127.0.0.1
```

Comment:
We'll actually use distributed nodes right away with Observer, and we'll see our golden_path remote node available when we do!